src/licensedcode/data/licenses/nysl-0.9982.yml

```diff
@@ -0,0 +1,10 @@
+key: nysl-0.9982
+short_name: NYSL 0.9982
+name: Software-License-NYSL-v0.9982
+category: Copyleft Limited
```

Comment:
@DennisClark  hey, can you please check the license category?

Reply:
@DennisClark any opinion?